internal/configuration/env_config.go

```diff
@@ -103,39 +105,39 @@ func (c *EnvConfig) GetPostgresPort() int {
 	return i
 }
 
-// GetPostgresUser returns the postgres user as set via default, config file, or environment variable
+// GetPostgresUser returns the postgres user as set via default, config file, or environment variable.
 func (c *EnvConfig) GetPostgresUser() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetPostgresDatabase returns the postgres database as set via default, config file, or environment variable
+// GetPostgresDatabase returns the postgres database as set via default, config file, or environment variable.
 func (c *EnvConfig) GetPostgresDatabase() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetPostgresPassword returns the postgres password as set via default, config file, or environment variable
+// GetPostgresPassword returns the postgres password as set via default, config file, or environment variable.
 func (c *EnvConfig) GetPostgresPassword() string {
 	callPtr, _, _, _ := runtime.Caller(0)
 	value := getConfigValueFromEnv(util.NameOfFunction(callPtr))
 
 	return value
 }
 
-// GetPostgresSSLMode returns the postgres sslmode as set via default, config file, or environment variable
+// GetPostgresSSLMode returns the postgres SSL mode as set via default, config file, or environment variable.
```

Comment:
it depends what is meant here. The flag is indeed `sslmode` in Postgres. No need to change imo.